llvm/lib/ObjectYAML/GOFFEmitter.cpp

```diff
@@ -259,6 +313,14 @@ bool GOFFState::writeObject() {
   writeHeader(Doc.Header);
   if (HasError)
     return false;
+  // Iterate over all records.
+  for (auto &Rec : Doc.Records) {
+    if (auto *Sym = dyn_cast<GOFFYAML::Symbol>(Rec.get())) {
```

Comment:
I think technically, according to the coding standards, this if/else shouldn't have braces.